server/store/datastore/migration/002_repos_drop_allow_deploys_allow_tags.go

```diff
@@ -26,4 +26,5 @@ var alterTableReposDropAllowDeploysAllowTags = task{
 			"repo_allow_tags",
 		)
 	},
+	dependency: []string{"xorm"},
```

Comment:
Wouldn't it be enough to say, migrations are executed in order an fail if one or more of the previous ones were skipped?

Reply:
currently if a migration fail, sql transaktion do a rollback and quit the program